pyplot/baselineFits/ModuleMapper.py

```diff
@@ -0,0 +1,134 @@
+#Module Mapper for 2019 layout
+def str_to_hw(string):
```

Comment:
Minor comment:
Performance is probably not an issue here, but, Python is a scripting language. So each time you call this function, it will reconstruct the entire string_to_hw array, and then find one item in it to return.
You would do much better creating a Python class which contains the string_to_hw translation table and then has a method (function) to do the conversion for a particular string. You can then have another method to do the reverse lookup.